components/brave_wallet_ui/components/desktop/wallet-onboarding/restore/style.ts

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components'
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  width: 100%;
+  padding-top: 50px;
+`
+
+export const Title = styled.span`
+  font-family: Poppins;
+  font-size: 20px;
+  font-weight: 600;
+  color: ${(p) => p.theme.color.text01};
+  letter-spacing: 0.02em;
+  margin-bottom: 8px;
+`
+
+export const Description = styled.span`
+  display: flex;
+  align-items: center;
+  font-family: Poppins;
+  font-size: 14px;
+  font-weight: 300;
+  color: ${(p) => p.theme.color.text02};
+  max-width: 520px;
+  text-align: center;
+  margin-bottom: 25px;
+`
+
+export const FormText = styled.span`
+  font-family: Poppins;
+  font-size: 15px;
+  font-weight: 600;
+  color: ${(p) => p.theme.color.text01};
+  letter-spacing: 0.04em;
+  margin-bottom: 8px;
+`
+
+export const FormWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  width: 100%;
+  max-width: 550px;
+`
+
+export const ErrorText = styled.span`
+  font-family: Poppins;
+  font-size: 12px;
+  line-height: 18px;
+  color: ${(p) => p.theme.color.errorText};
+  margin-bottom: 10px;
+`
+
+export const CheckboxRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  margin-left: -8px;
```

Comment:
negative margins make me nervous about overlapping something. Does this `-8px` correspond to an always-present padding area in its always-used parent?

Reply:
I think I added this because the `Checkbox` component from `brave-ui` has a padding set to it and i was unable to override the style.

Reply:
Fixed!